spec/spec_helper.rb

```diff
@@ -2,7 +2,10 @@
 require 'simplecov-rcov'
 SimpleCov.formatter = SimpleCov::Formatter::RcovFormatter
 SimpleCov.minimum_coverage 100
-
+SimpleCov.start do
+  add_filter "/vendor/bundle/"
```

Comment:
I still think this is the wrong thing, because it forces a particular location - presumably, your previous configuration of `.bundle` now won't work.